content/authors/a-theertha/_index.md

```diff
@@ -0,0 +1,6 @@
+---
+name:  A Theertha
+photo: '/authorImages/theertha.jpg'

```

Comment:
Question: Will this work without having the first forward slash? 
so like
photo: authors/images/...

Reply:
dont think it does, just tried it out